Source/FieldSolver/SpectralSolver/SpectralHankelTransform/HankelTransform.cpp

```diff
@@ -23,10 +25,18 @@ HankelTransform::HankelTransform (int const hankel_order,
 : m_nr(nr), m_nk(nr)
 {
 
+    WARPX_PROFILE("HankelTransform::HankelTransform");
+
     // Check that azimuthal_mode has a valid value
     WARPX_ALWAYS_ASSERT_WITH_MESSAGE(hankel_order-1 <= azimuthal_mode && azimuthal_mode <= hankel_order+1,
                                      "azimuthal_mode must be either hankel_order-1, hankel_order or hankel_order+1");
 
+#ifdef AMREX_USE_GPU
```

Comment:
Memo to ourselves: BLAS++ makes up their own SYCL device numbering and it might not fit ours.
Let's add a comment here?